lib/table.js

```diff
@@ -111,22 +111,33 @@ var Table = Class.extend({
             done(null, result);
         });
     },
-    _updateRecord: function(recordId, recordData, opts, done) {
-        var record = new Record(this, recordId);
-        if (!done) {
-            done = opts;
-            record.patchUpdate(recordData, done);
-        } else {
-            record.patchUpdate(recordData, opts, done);
-        }
-    },
-    _replaceRecord: function(recordId, recordData, opts, done) {
-        var record = new Record(this, recordId);
-        if (!done) {
-            done = opts;
-            record.putUpdate(recordData, done);
+    _updateRecords: function(isDestructiveUpdate, recordsOrRecordId, recordDataOrOptsOrDone, optsOrDone, done) {
```

Comment:
Should we rename to `recordsDataOrRecordId` for consistency with batch create? Same thing for `records` -> `recordsData`, since `records` sounds like `Array<Record>`. Or: `recordObjsOrRecordId` and `recordObjs`?